blog/2022-06-30-paralus-quickstart-aks/index.md

```diff
@@ -0,0 +1,163 @@
+
+<!--truncate-->
+
+**Table Of Content:**
+
+- [Pre Requisites](#pre-requisites)
+- [Creating AKS Cluster](#creating-aks-cluster)
+- [Installing Paralus](#installing-paralus)
+- [Configuring DNS Settings](#configuring-dns-settings)
+  - [Accessing The Dashboard](#accessing-the-dashboard)
+  - [Importing Existing Cluster](#importing-existing-cluster)
+
+## Pre Requisites
+
+To setup Paralus on Azure Kubernetes Service (AKS) there are a few prerequisites:
+
+- An Azure Account - _you can [register for a Free Azure account](https://azure.microsoft.com/en-in/free/) if you don't have one_
+- A Domain Name - _with permission to manage DNS settings._
+- Helm
+
+The logical steps would start with setting up a Kubernetes cluster on AKS. Deploying Paralus via helm charts followed by configuring DNS for your domain to work with Paralus. Lastly, logging into Paralus and importing a Kubernetes cluster.
+
+## Creating AKS Cluster
+
+One of the good things when it comes to creating an AKS cluster is the various options it gives you to create a cluster:
+
+- [Using the Azure CLI](https://docs.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-cli)
+- [Using the Azure PowerShell](https://docs.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-powershell)
+- [Using the Azure Portal](https://docs.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-powershell)
+- [Using the ARM template](https://docs.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-rm-template)
+
+You are free to choose any option that you want, we used the Azure Portal to create one.
+
+> Note: Choose a virtual machine with atleast 4 vCPUs and 16 GB of RAM. Refer to the list of [Azure VMs](https://azure.microsoft.com/en-in/pricing/details/virtual-machines/linux/).
+
+After the cluster is created, start the cluster and connect to it. If you've created a cluster using the Azure portal, you can use the Azure Cloud Shell to connect to the cluster.
+
+## Installing Paralus
+
+1. Add helm repo
+
+   `helm repo add paralus https://paralus.github.io/helm-charts`
+
+2. Install Paralus
+
+  ```bash
+   helm install myrelease paralus/ztka \
+    -f https://raw.githubusercontent.com/paralus/helm-charts/main/examples/values.aks.yaml \
+    -set fqdn.domain="chartexample.com" \
+    -n paralus \
+    --create-namespace
+  ```
+
+  >Note: If you plan to use Paralus without a `https` enabled domain, you'll need to set the `kratos.development` flas as `true`
+
+  ```bash
+   NAME: myrelease
+   LAST DEPLOYED: Wed Jun 29 10:13:48 2022
+   NAMESPACE: paralus
+   STATUS: deployed
+   REVISION: 1
+   NOTES:
+   1. Access the application URL by running these commands:
+     Get load balancer address via:
+     kubectl get service envoy --namespace paralus -o jsonpath='{.status.loadBalancer.ingress[0].hostname}'
+
+     Add DNS records of following domains such that it resolves to above address:
+     - console-aks-oss.chartexample.com
```

Comment:
Since we are not updating subdomains, Update these domains to
- console.chartexample.com
- *.core-connector.chartexample.com
- *.user.chartexample.com

Also, same change in DNS setting section below.

Reply:
Fixed